src/um/winnt.rs

```diff
@@ -2123,27 +2043,28 @@ STRUCT!{struct ACL_SIZE_INFORMATION {
     AclBytesFree: DWORD,
 }}
 pub type PACL_SIZE_INFORMATION = *mut ACL_SIZE_INFORMATION;
-pub const SECURITY_DESCRIPTOR_REVISION: DWORD = 1;
-pub const SECURITY_DESCRIPTOR_REVISION1: DWORD = 1;
-#[cfg(target_arch = "x86_64")]
-pub const SECURITY_DESCRIPTOR_MIN_LENGTH: usize = 40;
-#[cfg(target_arch = "x86")]
-pub const SECURITY_DESCRIPTOR_MIN_LENGTH: usize = 20;
+pub const SECURITY_DESCRIPTOR_REVISION: BYTE = 1;
+pub const SECURITY_DESCRIPTOR_REVISION1: BYTE = 1;
+#[cfg(target_arch="x86")]
+const __SIZE_OF_SECURITY_DESCRIPTOR: SIZE_T = 20;
+#[cfg(target_arch="x86_64")]
+const __SIZE_OF_SECURITY_DESCRIPTOR: SIZE_T = 40;
+pub const SECURITY_DESCRIPTOR_MIN_LENGTH: SIZE_T = __SIZE_OF_SECURITY_DESCRIPTOR;
 pub type SECURITY_DESCRIPTOR_CONTROL = WORD;
-pub type PSECURITY_DESCRIPTOR_CONTROL = *mut WORD;
-pub const SE_OWNER_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0001;
-pub const SE_GROUP_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0002;
-pub const SE_DACL_PRESENT: SECURITY_DESCRIPTOR_CONTROL = 0x0004;
-pub const SE_DACL_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0008;
-pub const SE_SACL_PRESENT: SECURITY_DESCRIPTOR_CONTROL = 0x0010;
-pub const SE_SACL_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0020;
+pub type PSECURITY_DESCRIPTOR_CONTROL = *mut SECURITY_DESCRIPTOR_CONTROL;
 pub const SE_DACL_AUTO_INHERIT_REQ: SECURITY_DESCRIPTOR_CONTROL = 0x0100;
-pub const SE_SACL_AUTO_INHERIT_REQ: SECURITY_DESCRIPTOR_CONTROL = 0x0200;
 pub const SE_DACL_AUTO_INHERITED: SECURITY_DESCRIPTOR_CONTROL = 0x0400;
-pub const SE_SACL_AUTO_INHERITED: SECURITY_DESCRIPTOR_CONTROL = 0x0800;
+pub const SE_DACL_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0008;
+pub const SE_DACL_PRESENT: SECURITY_DESCRIPTOR_CONTROL = 0x0004;
 pub const SE_DACL_PROTECTED: SECURITY_DESCRIPTOR_CONTROL = 0x1000;
-pub const SE_SACL_PROTECTED: SECURITY_DESCRIPTOR_CONTROL = 0x2000;
+pub const SE_GROUP_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0002;
+pub const SE_OWNER_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0001;
 pub const SE_RM_CONTROL_VALID: SECURITY_DESCRIPTOR_CONTROL = 0x4000;
+pub const SE_SACL_AUTO_INHERIT_REQ: SECURITY_DESCRIPTOR_CONTROL = 0x0200;
+pub const SE_SACL_AUTO_INHERITED: SECURITY_DESCRIPTOR_CONTROL = 0x0800;
+pub const SE_SACL_DEFAULTED: SECURITY_DESCRIPTOR_CONTROL = 0x0008;
+pub const SE_SACL_PRESENT: SECURITY_DESCRIPTOR_CONTROL = 0x0010;
+pub const SE_SACL_PROTECTED: SECURITY_DESCRIPTOR_CONTROL = 0x2000;
 pub const SE_SELF_RELATIVE: SECURITY_DESCRIPTOR_CONTROL = 0x8000;
```

Comment:
The old order of these constants was correct. Why did you reorder them to a different order than what is in the header?